docs/src/misc.rst

```diff
@@ -289,6 +303,23 @@ API
 
     .. versionadded:: 1.9.0
 
+.. c:function:: int uv_get_passwd(uv_passwd_t* pwd)
+
+    Gets a subset of the password file entry for the current effective user.
+    The populated data includes the username, uid, gid, shell, and home
+    directory. On non-Windows systems, all data comes from `getpwuid_r()`. On
+    Windows, uid and gid have no meaning, and shell is `NULL`. After
+    successfully calling this function, the memory allocated to `pwd` needs to
+    be freed with :c:func:`uv_free_passwd`.
```

Comment:
Can you mention here that the effective UID, not the real UID, is used for looking up the passwd record?